governance/votes/Executive vote - October 04, 2024.md

```diff
@@ -0,0 +1,158 @@
+- Disable the PSM-USDC-A DC-IAM.
+
+#### LITE-PSM-USDC-A Buffer Increase
+
+- Increase the [LITE-PSM-USDC-A Buffer (`buf`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.6_Buffer_Definition-cf8e3d20-fffa-4b6a-9bf2-169d493c3638|57ea599773d45150b2632df9) by 200 million DAI from 200 million DAI to **400 million DAI**.
+
+#### LITE-PSM-USDC-A DC-IAM Update
+
+- Increase the [Maximum Debt Ceiling (`line`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.3_Maximum_Debt_Ceiling_Definition-e4a0d8f1-4a01-46a7-b693-a94e7f4afc1a|57ea599773d45150b2632df9) by 2,5 billion DAI from 7,5 billion DAI to **10 billion DAI**.
+- Increase the [Target Available Debt (`gap`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.4_Target_Available_Debt_Definition-b876ecf4-a901-4721-9c4b-9f2fc21f954c|57ea599773d45150b2632df9) by 200 million DAI from 200 million DAI to **400 million DAI**.
+- Keep the [Ceiling Increase Cooldown (`ttl`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.5_Ceiling_Increase_Cooldown_Definition-3d5980c5-7036-4b68-93e0-a48551fe09ed|57ea599773d45150b2632df9) unchanged at **12 hours**.
+
+### Aave Lido Market Spark USDS Direct Deposit Module (D3M) Activation
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/risk-assessment-and-parameter-recommendations-spark-ddm-to-aave-lido-market/25175/2)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/risk-assessment-and-parameter-recommendations-spark-ddm-to-aave-lido-market/25175)
+
+If this executive proposal passes, then the Aave Lido Market Spark USDS D3M will be activated by executing the following actions:
+
+- Add new ilk as DIRECT-SPK-AAVE-LIDO-USDS.
+- Set up the D3M DC-IAM.
+- Set up the D3M addresses.
+- Extend the [DIRECT_MOM Breaker](https://sky-atlas.powerhouse.io/#A.1.8.2.2.5_Direct_Deposit_Breaker_Exception-c240d4fa-ab20-4787-8dbf-7c94c8e78347%7C0db30758e055352c) to include the new D3M.
+
+These actions are further described below.
+
+#### New ilk Addition
+
+A new ilk will be added as DIRECT-SPK-AAVE-LIDO-USDS.
+
+#### Aave Lido Market Spark USDS D3M Setup
+
+- [Maximum Debt Ceiling (`line`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.3_Maximum_Debt_Ceiling_Definition-e4a0d8f1-4a01-46a7-b693-a94e7f4afc1a|57ea599773d45150b2632df9): 100 million USDS.
+- [Target Available Debt (`gap`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.4_Target_Available_Debt_Definition-b876ecf4-a901-4721-9c4b-9f2fc21f954c|57ea599773d45150b2632df9): 50 million USDS.
+- [Ceiling Increase Cooldown (`ttl`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.5_Ceiling_Increase_Cooldown_Definition-3d5980c5-7036-4b68-93e0-a48551fe09ed|57ea599773d45150b2632df9): 24 hours.
+- Debt Write-off Timelock (`tau`) (Defines the time to elapse before debt can be written off after a D3M has been caged): 7 days.
```

Comment:
note to @VoteWizard - let's ask @Le-Bateleur-2023 to get this added to the atlas.